From 756613c76f4e7d238d06fb092f1b6625f9c83350 Mon Sep 17 00:00:00 2001
From: Sliim <sliim@mailoo.org>
Date: Sun, 31 Mar 2013 19:28:52 +0200
Subject: [PATCH] Personal changes.

---
 .gitignore           |    7 ++++++-
 modules/prelude-c.el |    2 +-
 2 files changed, 7 insertions(+), 2 deletions(-)

diff --git a/.gitignore b/.gitignore
index 12fa4ba..e254e68 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,5 +1,10 @@
 *~
 *.elc
+/dvc
+/personal
+/php-manual
+/emms
+/ac-comphist.dat
 auto-save-list
 recentf
 savehist
@@ -16,4 +21,4 @@ places
 .smex-items
 savefile/
 /prelude-modules.el
-projectile-bookmarks.eld
\ No newline at end of file
+projectile-bookmarks.eld
diff --git a/modules/prelude-c.el b/modules/prelude-c.el
index ab1a78b..330e610 100644
--- a/modules/prelude-c.el
+++ b/modules/prelude-c.el
@@ -35,7 +35,7 @@
 (require 'prelude-programming)
 
 (defun prelude-c-mode-common-defaults ()
-  (setq indent-tabs-mode t)
+  (setq indent-tabs-mode nil)
   (setq c-basic-offset 4))
 
 (setq prelude-c-mode-common-hook 'prelude-c-mode-common-defaults)
-- 
1.7.10.4

